test/foursquare-lunch-test.coffee

```diff
@@ -1,21 +1,34 @@
+# Note: You must set up the following variables inside a .env file in the root
+# of the project to test the foursquare response:
+#   FOURSQUARE_CLIENT_ID
+#   FOURSQUARE_CLIENT_SECRET
+#   FOURSQUARE_ACCESS_TOKEN
+#   HUBOT_DEFAULT_LATITUDE
+#   HUBOT_DEFAULT_LONGITUDE
```

Comment:
:heart: the test improvements! So for these, I'd rather use some fake credentials and [`nock`](https://github.com/node-nock/nock) to simulate the "live" calls. It's a bit of a hassle to set up, but it produces reliable results based on what a "known good" response looks like. It would also let us account for cases where the configuration is set.